### ISSUE TITLE
Define minimum required element set for description of each iteration (revision)

### ISSUE THREAD

**Issue by regineheberlein** [saa-ts-dacs/dacs#73]
This relates to [Required Elements](https://saa-ts-dacs.github.io/dacs/06_part_I/02_chapter_01.html) as affected by [Principle 11](https://saa-ts-dacs.github.io/dacs/04_statement_of_principles.html#11--archival-description-is-a-continuous-intellectual-endeavor)

## Describe how DACS does not currently meet this principle

There is currently sparse guidance on how minor edits or major revisions to the description should be documented.

Documenting a major revision should be required. What constitutes a major revision should be defined to some degree: any substantive change to identifying elements, any change that re-contextualizes the material or entity, or any other change that has the potential to affect the discoverability or meaning of the material or entity should be documented.

The documentation should include the following:
- a structured revision statement, with parsable data elements for the author of the change, the nature of the change, and the date of the change
- a narrative processing note explaining the change
- updates to rights statements
- updates to access statements

The guidance should also include that all of the above information should be added onto for each iteration; it should never be overwritten but persist as a record of the changes over time.

## I expect that this will require a

- [ ] Minor change to DACS
- [x] Major change to DACS

**Comment by searcy** [comment]
I think this is a great idea that supports not only Principle 11 but also Principle 6. Upon reading it for the first time, though, I was a little unclear as to whether this issue was meant to provide instruction for the descriptive elements archivists should be reviewing and editing in the course of furthering their descriptive work, or if it to identify how concretely archivists should document those revisions. The former will always be different depending on the collection and the circumstances that prompted the revision, whereas the latter is more consistent. I assume the ticket is for the latter interpretation, but I wanted to point out that ambiguity.

In addition to the elements suggested above, I would also point to 8.1.5 and perhaps 4.6 if there is a mechanism for users to access a previous version of the finding aid, along with some guidance as to what constitutes a substantial change.

**Comment by regineheberlein** [comment]
Yes, in re-reading this, I take your point! Trying to reframe this a bit better: I suggest guidance for iterative processing should be:

- the description created in the first first iteration should include everything defined as the the single- or multi-level minimum requirements, respectively
-  each subsequent iteration should include, at a minimum, 
    - a revision note stating changes to the description
    - a processing note stating changes to the physical state of the material
    - a rights review
    - an access review

So this issue seeks to define what pieces of information should be captured during each iteration of the iterative processing cycle; whereas #70 more narrowly defines the revision note as one of those pieces.
